enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion.unit.spec.tsx

```diff
@@ -83,9 +83,7 @@ describe("InteractiveQuestion", () => {
     expect(screen.getByTestId("loading-spinner")).toBeInTheDocument();
   });
 
-  // TODO [Oisin]: fix failing test in "Fix Interactive Question" PR
-  // eslint-disable-next-line jest/no-disabled-tests
-  it.skip("should render when question is valid", async () => {
```

Comment:
This PR also fix this test. I try running with the previous version and it's still fail